<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddrinfoWindow</class>
 <widget class="QMainWindow" name="AddrinfoWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>456</width>
    <height>225</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Client</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>321</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>IP Address</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="Addr0LineEdit">
           <property name="text">
            <string>127</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>.</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="Addr1LineEdit">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>.</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="Addr2LineEdit">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>.</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="Addr3LineEdit">
           <property name="text">
            <string>1</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Port</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="PortLineEdit">
         <property name="text">
          <string>6666</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>170</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Ok</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>pushButton</tabstop>
  <tabstop>Addr0LineEdit</tabstop>
  <tabstop>Addr1LineEdit</tabstop>
  <tabstop>Addr2LineEdit</tabstop>
  <tabstop>Addr3LineEdit</tabstop>
  <tabstop>PortLineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
